scratch.el

```diff
@@ -36,3 +36,20 @@
     (unless buffer
       (setf buffer "BUFFER NAME")))
   (oref room extra))
+
+;;; derp
+
+(cl-defmethod matrix-client-ng-room-command-cowsay ((room matrix-room) input)
+  "Cowsay!"
+  (let* ((s (replace-regexp-in-string (rx bos "/" (1+ (not space)) (1+ space)) "" input))
+         (cow-type (seq-random-elt '("-b" "-d" "-g" "-p" "-s" "-t" "-w" "-y")))
+         (cowsaid (shell-command-to-string (format "cowsay %s %s" cow-type (shell-quote-argument s))))
+         (html (concat "<font color=\"#000\" data-mx-color=\"#000\" data-mx-bg-color=\"#fff\"><pre>\n"
```

Comment:
If we end up doing more of this, maybe we should write a function to stick the string into a buffer and htmlize it (I don't think there's a working function for doing this in htmlize already). Right now though, this is probably simpler.